content/hacking-atom/sections/debugging.md

```diff
@@ -297,3 +297,28 @@ You can then include the startup profile in any Issue you report.
 If you are having issues installing a package using `apm install`, this could be because the package has dependencies on libraries that contain native code. This means you will need to have a C++ compiler and Python installed to be able to install it. You can run `apm install --check` to see if the Atom package manager can build native code on your machine.
 
 Check out the pre-requisites in the [build instructions](https://github.com/atom/atom/tree/master/docs/build-instructions) for your platform for more details.
+
+#### Check if your GPU is causing the problem
+
+If you encounter flickering or other rendering issues, you can disable your Graphics Processing Unit (GPU) with the `--disable-gpu` Chromium flag to see if the fault lies with your GPU:
```

Comment:
> disable your Graphics Processing Unit

I think this might be better worded as something like "stop Atom from using your Graphics...". The current wording might imply that it _completely_ disables it for everything.

----

💭 Should this just say "the following flag"?